internal/backends/postgresql/metadata/registry_test.go

```diff
@@ -147,49 +148,6 @@ func TestCheckAuth(t *testing.T) {
 	}
 }
 
-func TestCreateDrop(t *testing.T) {
```

Comment:
I deleted this test because it's a simple test that only tests simple situations when we create and drop collections/databases, and this is what we do in every integration tests.